moveit.rosinstall

```diff
@@ -12,10 +12,6 @@
     local-name: moveit_resources
     uri: https://github.com/ros-planning/moveit_resources.git
     version: master
-- git:
-    local-name: moveit_visual_tools
-    uri: https://github.com/ros-planning/moveit_visual_tools.git
-    version: melodic-devel
```

Comment:
why? of course im biased but i think this is valuable to keep here

Reply:
This was neccessary (for now) as you? didn't released `graph_msgs`.

Reply:
makes sense. can you just comment it out then?

im having @davesteffen get up to speed on releasing ROS packages with bloom, and this is first on his list but he's still learning

Reply:
Yes, I will comment-out only.